src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,68 +2,70 @@
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
-import java.util.ArrayList;
-import java.util.List;
 
 public class SalaryInfo {
-
+    private static final int DATE_IN_DATA_POSITION = 0;
+    private static final int NAME_IN_REPORT_POSITION = 0;
+    private static final int SALARY_RATE_PER_HOUR_POSITION = 3;
+    private static final int TOTAL_SALARY_IN_REPORT_POSITION = 2;
+    private static final int WORKED_HOURS_POSITION = 2;
+    private static final int ZERO_VALUE = 0;
     private static final String DATE_FORMAT = "dd.MM.yyyy";
     private static final String MINUS_DELIMITER = "-";
     private static final String NEW_STRING_DELIMITER = "\n";
     private static final String SPACE_DELIMITER = " ";
     private static final String TITLE_OF_REPORT = "Report for period";
+    private static final String INSERT_BETWEEN_VALUES = SPACE_DELIMITER + MINUS_DELIMITER
+            + SPACE_DELIMITER;
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
         LocalDate startOfPeriod = LocalDate.parse(dateFrom, formatter);
         LocalDate endOfPeriod = LocalDate.parse(dateTo, formatter);
-
         if (!startOfPeriod.equals(endOfPeriod)) {
             endOfPeriod = endOfPeriod.plusDays(1);
         }
-
-        List<Employee> listEmployee = new ArrayList<>();
-
-        for (String name : names) {
-            listEmployee.add(new Employee(name, 0));
+        String[] report = new String[names.length];
+        StringBuilder stringBuilder = new StringBuilder();
+        for (int i = 0; i < names.length; i++) {
+            report[i] = stringBuilder.append(names[i]).append(INSERT_BETWEEN_VALUES)
+                                     .append(ZERO_VALUE).toString();
+            stringBuilder.setLength(ZERO_VALUE);
```

Comment:
further in the code you are rewriting this

Reply:
I agree. Removed.